keylime/tenant.py

```diff
@@ -103,6 +103,9 @@ def __init__(self):
         self.api_version = keylime_api_version.current_version()
         self.enable_agent_mtls = config.getboolean("tenant", "enable_agent_mtls")
         self.request_timeout = config.getint("tenant", "request_timeout", fallback=60)
+        self.retry_interval = config.getfloat("tenant", "retry_interval")
+        self.exponential_backoff = config.getboolean("tenant", "exponential_backoff")
+        self.maxr = config.getint("tenant", "max_retries")
```

Comment:
Are those new options? Should we do a config migration? Add some defaults?

Reply:
There were already there, so no migration or default options needed: https://github.com/keylime/keylime/blob/9dae00e519ba5f37cc7b8bfbebdb2ad6953f03bc/templates/2.0/tenant.j2#L78